libs/s3/objects.go

```diff
@@ -77,3 +78,39 @@ func (s Client) Load(i interface{}, path string) error {
 
 	return nil
 }
+
+// PutObject takes a byte array and saves it in S3 with the given key name
+func (s Client) PutObject(object []byte, key string, ct string) error {
```

Comment:
```golang
func (s Client) PutObject(object []byte, key, ct string) error {
```
probably use `contentType` as the variable name too - its more descriptive

Reply:
it might be more intuitive to have the order changed too -- `ket, contentType, object`